electron/gog/user.ts

```diff
@@ -48,7 +48,10 @@ export class GOGUser {
     const user: GOGLoginData = configStore.get('credentials') as GOGLoginData
     const response = await axios
       .get(`https://embed.gog.com/userData.json`, {
-        headers: { Authorization: `Bearer ${user.access_token}` }
+        headers: {
+          Authorization: `Bearer ${user.access_token}`,
+          'User-Agent': 'GOGGalaxyClient/2.0.45.61 (GOG Galaxy)'
```

Comment:
The user agent doesn't seem necessary (when I made a request to the above `userData.json`), any reason we spoof it to be GOG Galaxy, we could have it be GogDL or nothing at all